PrusaSlicer-2.5-M5-Profile/vendor/AnkerMake.ini

```diff
@@ -419,7 +419,7 @@ wipe = 1
 z_offset = 0
 default_filament_profile = "Generic PLA+ @ANKER"
 start_gcode = M104 S{first_layer_temperature[0]} ; set final nozzle temp\nM190 S{first_layer_bed_temperature[0]} ; set and wait for nozzle temp to stabilize\nM109 S{first_layer_temperature[0]} ; wait for nozzle temp to stabilize\nG28 ;Home\nG1 E10 F3600; push out retracted filament(fix for over retraction after prime)
-end_gcode = M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84
+end_gcode = M104 S0\nM140 S0\n;Move nozzle 8mm up from print\nG91\nG1 Z8\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84
```

Comment:
You can do logic in Prusa Slicer but not Cura. Something like:
`{if max_layer_z < 40}\nG91\nG1 Z8 F600 ; Move print head up{endif}`
I have yet to test this. Just read about it.